test/app/languageforge/lexicon/shared/editor.page.ts

```diff
@@ -309,7 +309,7 @@ export class EditorPage {
 
     bubbles: {
       first: element.all(by.css('.dc-entry .commentBubble')).get(1),
-      second: element.all(by.css('.dc-entry .dc-sense .commentBubble')).get(0)
+      second: element.all(by.css('.dc-entry .dc-sense .commentBubble')).get(1)
```

Comment:
I know this was merged already, but if you notice this comment and have time to satisfy my curiosity can you explain why this line was affected?